install_config/aggregate_logging.adoc

```diff
@@ -4607,7 +4632,7 @@ logs to determine the source of the problem.
 
 *Image pull fails*
 
-If you specify an `openshift_logging_image_prefix` that results in images being defined
+If you specify an `openshift_logging_image` that results in images being defined
```

Comment:
I suggest to remove this variable. In v3.11, oreg_url can be used to indicate the image prefix for logging.